### PR TITLE
Add better description of pressure dependency

### PR DESCRIPTION
**Pull request by eivindjahren**

### CODE REVIEW

File: docs/adusting_with_depth_and_pressure.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+The dry_rock model can be adjusted according to depth and pressure:
+
+```yaml
+dry_rock:
+    model:
+        # model type is one of:
+        #  * "polyfit"
+        #  * "friable_sand"
+        #  * "patchy_cement"
+        type: "polyfit"
```
Comment (kwinkunks) [inline docs/adusting_with_depth_and_pressure.md:13]: Is it idiomatic to use quotes for strings in this library? Seems a bit noisy?

File: docs/adusting_with_depth_and_pressure.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+         coefficients:
+           # adjusted_bulk_modulus = c[i][j] * delta_depth^i * original_bulk_modulus^j
+           bulk_modulus: [[0.0, 1.5], [1.0, 0.0]]
+           # adjusted_shear_modulus = c[i][j] * delta_depth^i * original_shear_modulus^j
+           shear_modulus: [[0.0, 1.2], [1.0, 0.0]]
+           # adjusted_density = c[i][j] * delta_depth^i * original_density^j
+           density: [[0.0, 1.35], [1.0, 0.0]]
+```
+
+In the above example, the dry_rock material changes both with depth and pressure according to
```
Comment (kwinkunks) [inline docs/adusting_with_depth_and_pressure.md:64]: Might be nice to use backticks for the names of things,eg `dry_rock`, `pressure_dependency`, etc. Personally I find this helps read docs with a lot of names floating about.

File: docs/adusting_with_depth_and_pressure.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+the "depth_trend" and "pressure_dependency" adjustment. Depth trend simply adjusts the material 
+according to a two dimensional polynomial (see the above example for details).
```
Comment (kwinkunks) [inline docs/adusting_with_depth_and_pressure.md:66]: Maybe just use one line per paragraph and let everything wrap? Would prefer to the mixture of line breaks and wrapped lines. But whatever is currently usual in this library I guess.

File: docs/adusting_with_depth_and_pressure.md (hunk 1 above)
Comment (eivindjahren) [reply docs/adusting_with_depth_and_pressure.md:13]: Right, I'll do away with those.

File: docs/adusting_with_depth_and_pressure.md (hunk 3 above)
Comment (eivindjahren) [reply docs/adusting_with_depth_and_pressure.md]: I used linewrap everywhere else, and I prefer it that way unfortunately :melting_face: . I have worked in some projects that use linewrap for natural text and personally I really didn't like it.

**Review by kwinkunks — APPROVED** [review]
Nice!

**Pull request merged**